pints/_mcmc/_adaptive_covariance.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, x0, sigma0=None):
         # Acceptance rate monitoring
         self._iterations = 0
         self._acceptance = 0
+        self._accepted_count = 0
```

Comment:
I don't like how this is split up: Why is this variable defined here, but only used by a different class?

Reply:
Ok. Have moved these to the daughter class since this is where they are used.